modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/paths/Testray.path

```diff
@@ -12,12 +12,17 @@
 <tbody>
 <tr>
 	<td>ITEMS_PER_PAGE_BUTTON</td>
-	<td>//div[contains(@class, testray-page)]//div[contains(@class, 'pagination')]//button[contains(@class, 'dropdown')]</td>
+	<td>xpath=(//div[contains(@class, 'pagination')]//button[contains(@class, 'dropdown') and contains(.,'')])[3]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>ITEMS_PER_PAGE_BUTTON_IN_PROJECTS</td>
+	<td>xpath=(//div[contains(@class, 'pagination')]//button[contains(@class, 'dropdown') and contains(.,'')])[1]</td>
 	<td></td>
 </tr>
 <tr>
 	<td>ITEMS_PER_PAGE_BUTTON_DROPDOWN</td>
-	<td>//div[contains(@class, 'dropdown-menu')]//button[(text()='${key_itemValue}')]</td>
+	<td>//div[contains(@class, 'dropdown-menu show')]//button[contains(@class, 'dropdown-item') and contains (.,'150 Items')]</td>
 	<td></td>
```

Comment:
Is there a way for us to create a path for items per page to accept the other values, and not only 150 Items?